frontend/packages/dev-console/src/components/project-access/RenderProjectAccessPage.tsx

```diff
@@ -8,7 +8,9 @@ export interface RenderProjectAccessPageProps {
   namespace: string;
 }
 
-const RenderProjectAccessPage: React.FC<RenderProjectAccessPageProps> = ({ namespace }) => {
+export const RenderProjectAccessPageComponent: React.FC<RenderProjectAccessPageProps> = ({
```

Comment:
Change name back to RenderProjectAccessPage once the default export is reverted.